README.md

```diff
@@ -74,7 +74,7 @@ Here's what you need to do to get `iotedgedev` running on your dev machine. If y
     
 
 1. Install **Python 2.7 or Python 3**
-    - Windows - [Install from Python's website](https://www.python.org/downloads/)
+    - Windows - [Install from Python's website, use Win64 installer](https://www.python.org/downloads/)
```

Comment:
Are we saying the tool on works on x64 machines?  I'm not sure that is a requirement.  If not, you can remove this and let the user decide based on their machine arch.